packages/graphql/src/mutations/_txHelper.js

```diff
@@ -55,6 +55,22 @@ function useRelayer({ mutation, value }) {
   return true
 }
 
+/**
+ * trigger the EVM to mine a block, supporting the WPE-style providers, and
+ * the standard ones...
+ * @param web3 {Web3} a Web3 instance
+ */
+async function mineBlock(web3Inst) {
+  if (typeof web3Inst.currentProvider.sendAsync !== 'undefined') {
+    return await web3Inst.currentProvider.sendAsync(
+      { method: 'evm_mine' },
+      () => {}
+    )
+  } else {
+    return web3Inst.currentProvider.send({ method: 'evm_mine' }, () => {})
+  }
+}
+
```

Comment:
Do we need the async / await here? Or could we do something like this:
```js
function mineBlock(web3Inst) {
  const hasAsync = typeof web3Inst.currentProvider.sendAsync !== 'undefined'
  const sendMethod = hasAsync ? 'sendAsync' : 'send'
  return web3Inst.currentProvider[sendMethod]({ method: 'evm_mine' }, () => {})
}
```

Reply:
yeah, that should work as well.  Doesn't look like anything is dependent on it returning anything.  Will have that change up a little later this morning.